docs/docs.less

```diff
@@ -169,7 +158,6 @@ min-version {
 // ----- BANNER ---------------
 // currently used by test page layout to display a warning to readers
 .banner {
-  .sticky(vertical);
```

Comment:
Removed sticky styling of the warning banner that appears on test pages. It was getting in the way of many test cases, so it now just sits at the very top of the page, so that it's the first content someone would see if they navigate to a test page.